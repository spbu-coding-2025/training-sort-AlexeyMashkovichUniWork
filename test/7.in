652121565 981989647 -275624709 -367662412 302834583 346718723 774796402 1003416665 -112028975 -381210564 1047059575 -14628529 145882954 871909653 -874920965 83601066 88332616 -854406081 940662453 957374478 286131562 557983849 116547274 574349006 969177016 518349204 31807619 -527242370 -974539163 -1010400101 396058911 751324225 1045331369 -953307623 -690080012 274424128 467152924 -989025434 204098968 -718617876 -296494175 177416718 340495419 923130602 -24415453 539316277 -67010157 -1009824662 758652019 -200089528 1021291639 -28958243 -715847504 64097088 -528351062 -820412313 -491295532
