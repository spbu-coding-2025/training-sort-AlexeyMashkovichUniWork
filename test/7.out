-1010400101 -1009824662 -989025434 -974539163 -953307623 -874920965 -854406081 -820412313 -718617876 -715847504 -690080012 -528351062 -527242370 -491295532 -381210564 -367662412 -296494175 -275624709 -200089528 -112028975 -67010157 -28958243 -24415453 -14628529 31807619 64097088 83601066 88332616 116547274 145882954 177416718 204098968 274424128 286131562 302834583 340495419 346718723 396058911 467152924 518349204 539316277 557983849 574349006 652121565 751324225 758652019 774796402 871909653 923130602 940662453 957374478 969177016 981989647 1003416665 1021291639 1045331369 1047059575
